apps/pl-default/pattern-lab/alter-twig.php

```diff
@@ -192,7 +192,7 @@ function addFilters(\Twig_Environment &$env, $config) {
         }
         $renderable_attributes[] = "{$key}=\"{$value}\"";
       }
-      return implode($renderable_attributes, ' ');
+      return implode(' ', $renderable_attributes);
```

Comment:
@yuginaili `implode` is also used above on line 191.

Reply:
Thanks @jryanconklin